Assets/GothicVR/Scripts/Manager/UIManager.cs

```diff
@@ -0,0 +1,37 @@
+using System.Linq;
+using GVR.Creator;
+using GVR.Phoenix.Interface;
+using GVR.Util;
+using GVR.Caches;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+using UnityEngine.XR.Interaction.Toolkit;
+using UnityEngine.Events;
+
+namespace GVR.Manager
+{
+    public class UIManager : SingletonBehaviour<UIManager>
+    {
+        public Material GothicLoadingMenuMaterial;
+        public Material LoadingBarBackgroundMaterial;
+        public Material LoadingBarMaterial;
+        public void LoadDefaultTextures()
+        {
+            var loadingBackgroundTexture = AssetCache.I.TryGetTexture("LOADING.TGA");
+            GothicLoadingMenuMaterial.mainTexture = loadingBackgroundTexture;
```

Comment:
Now I get it with the materials being used.
Nevertheless I would challenge one more thing: Do the materials differ on their settings in the Asset folder? If so, how much? i.e. if there are more than 3 settings you need to do for the materials, we should definitely have them on assets. If not... I will probe again. ;-)

Reply:
Makes sense, we can create the material in script, seems to be a better idea

Reply:
Removed the materials and using script generated materials